src/index.html

```diff
@@ -16,17 +16,28 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-wrapper">
+    <form
+      class="search-container"
+      data-qa="big"
+    >
+      <input
+        class="search-bar search-bar__big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
The class name 'search-bar__big' suggests that you might be trying to represent a modifier for 'search-bar', which is a BEM naming convention. However, 'search-bar' itself is also used as a class here. If 'search-bar__big' is indeed a modifier, consider using 'search-bar search-bar--big' instead.